backend/src/main/java/de/jankahle/capstone/utility/ImageUtility.java

```diff
@@ -0,0 +1,20 @@
+package de.jankahle.capstone.utility;
+
+public class ImageUtility {
+    private static final String[] validTypes = {
+            "image/gif",
+            "image/png",
+            "image/jpg",
+            "image/jpeg",
+            "image/pjpeg"};
+
+    public static boolean IsContentTypeValid(String contentType) {
+        if (contentType.isEmpty())
+            return false;
+        for (String validType : validTypes) {
+            if (contentType.equals(validType))
+                return true;
+        }
+        return false;
```

Comment:
What do you think about this? Using a list makes it look much nicer imho.

```suggestion
    private static final List<String> validTypes = List.of(
            "image/gif",
            "image/png",
            "image/jpg",
            "image/jpeg",
            "image/pjpeg");

    public static boolean IsContentTypeValid(String contentType) {
        return !contentType.isEmpty() && validTypes.contains(contentType);
```